packages/commonwealth/test/mocha-dev.opts

```diff
@@ -2,4 +2,6 @@
   --require tsconfig-paths/register
   --require @babel/register
   --require source-map-support/register
+  --require ignore-styles
```

Comment:
required to get tests to run, because there is CSS being used in the file for finding breakpoints

**Without**

```
$ NODE_ENV=test ts-mocha --project tsconfig.json './test/unit/**/*.spec.ts'
Applied changes: logLevel=Debug to log groups 'all'.

/Users/dillchen/Coding/hicommonwealth/commonwealth/packages/commonwealth/node_modules/ts-mocha/node_modules/ts-node/src/index.ts:261
    return new TSError(diagnosticText, diagnosticCodes)
           ^
TSError: ⨯ Unable to compile TypeScript:
client/styles/mixins/breakpoints.scss(2,22): error TS1005: ';' expected.
client/styles/mixins/breakpoints.scss(3,23): error TS1005: ';' expected.
client/styles/mixins/breakpoints.scss(4,23): error TS1005: ';' expected.
client/styles/mixins/breakpoints.scss(5,29): error TS1005: ';' expected.
client/styles/mixins/breakpoints.scss(6,29): error TS1005: ';' expected.
client/styles/mixins/breakpoints.scss(7,22): error TS1005: ';' expected.
client/styles/mixins/breakpoints.scss(8,22): error TS1005: ';' expected.
client/styles/mixins/breakpoints.scss(9,28): error TS1005: ';' expected.
client/styles/mixins/breakpoints.scss(12,25): error TS1005: ';' expected.
client/styles/mixins/breakpoints.scss(12,52): error TS1351: An identifier or keyword cannot immediately follow a numeric literal.

```
See similar issues on [StackOverflow](https://stackoverflow.com/questions/32236443/mocha-testing-failed-due-to-css-in-webpack)


**With**
You'll see the current file